library/src/main/scala/response/cookies.scala

```diff
@@ -2,11 +2,6 @@ package unfiltered.response
 
 import unfiltered.Cookie
 
-/** Set-Cookie response header. See SetCookies */
-object ResponseCookies {
```

Comment:
nice catch :) `object ResponseCookies` does not have `apply` method. Should I fix `docs/09/b.markdown` like this?

```diff
--- a/docs/09/b.markdown
+++ b/docs/09/b.markdown
@@ -22,10 +22,10 @@ unfiltered.filter.Plan {
       })
     case Path("/prefer") & Params(p) =>
        // let's store it on the client
-       ResponseCookies(Cookie("pref", p("pref")(0))) ~>
+       SetCookies(Cookie("pref", p("pref")(0))) ~>
          Redirect("/")
     case Path("/forget") =>
-       ResponseCookies(Cookie("pref", "")) ~>
+       SetCookies(Cookie("pref", "")) ~>
          Redirect("/")
```